brouter-routing-app/src/main/java/btools/routingapp/BRouterWorker.java

```diff
@@ -57,7 +57,7 @@ public String getTrackFromParams(Bundle params) {
       if ("osmand".equalsIgnoreCase(tiFormat)) {
         rc.turnInstructionMode = 3;
       } else if ("locus".equalsIgnoreCase(tiFormat)) {
-        rc.turnInstructionMode = 2;
+        rc.turnInstructionMode = 7;
       }
```

Comment:
Are we not missing OsmAnd here?

Reply:
How do you mean that? OsmAnd is still present. Only timode for locus is switched to new standard output.

Reply:
You are completely right, not sure what I saw.